src/main/java/com/endava/Application.java

```diff
@@ -3,17 +3,32 @@
 import java.io.IOException;
 
 public class Application {
+    public static String path1 = "./src/main/resources/menu1.csv";
+    public static String path2 = "./src/main/resources/menu2.csv";
 
     public static void main(String[] args) throws IOException {
         double start = System.currentTimeMillis();
 
-        double menu1AveragePrice = FileUtil.getPriceAverage("./src/main/resources/menu1.csv");
-        double menu2AveragePrice = FileUtil.getPriceAverage("./src/main/resources/menu2.csv");
+        double menu1AveragePrice = FileUtil.getPriceAverage(path1);
+        double menu2AveragePrice = FileUtil.getPriceAverage(path2);
 
         double end = System.currentTimeMillis();
 
         System.out.println("Menu 1 average price: " + menu1AveragePrice);
         System.out.println("Menu 2 average price: " + menu2AveragePrice);
         System.out.println("Time to process: " + (end - start) / 1000 + " seconds");
+
+        start = System.currentTimeMillis();
+
+        Thread thread1 = new Thread(new RunnableImpl(path1), "Menu 1 average price: ");
+        Thread thread2 = new Thread(new RunnableImpl(path2), "Menu 2 average price: ");
+        System.out.print(thread1.getName());
+        thread1.run();
```

Comment:
You should use the start method. Using start() will create an actual thread of execution in the operating system and will execute what's inside the run method.